generator/src/test/java/com/scottlogic/deg/generator/cucumber/MatchingRegex.feature

```diff
@@ -466,44 +466,66 @@ Scenario: Running a 'matchingRegex' request alongside a granularTo constraint sh
       | "10" |
 
 Scenario: Running a 'matchingRegex' request alongside a after constraint should be successful
-     Given foo is matching regex /[0-z]{2}/
+     Given foo is matching regex /[a-z]{2}/
        And foo is after 2018-10-10T00:00:00.000
-     Then the following data should be included in what is generated:
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
         | foo  |
-        | null |
```

Comment:
Why is null no longer generated?

Reply:
This is a defect, I'll raise it accordingly.